app/api/games/route.tsx

```diff
@@ -1,22 +1,24 @@
-import prisma from "@/prisma/prisma";
+import prisma from "@/app/api/games/validation";
 
-export async function POST(req: Request) {
-	const body = await req.json();
+export async function POST(request: Request) {
+  const body = await request.json();
 
-	const result = await prisma.game.create({
-		data: {
-			...body,
-		},
-	});
-	if (!result)
-		return Response.json({
-			message: "error",
-			status: 500,
-		});
-	return Response.json({ message: "ok", status: 200, data: result });
+  try {
+    const result = await prisma.game.create({
+      data: {
+        ...body,
+      },
+    });
+
+    return new Response(JSON.stringify({ ...result }), { status: 201 });
```

Comment:
I don't think spreading `result` object is necessary (?)

```suggestion
    return new Response(JSON.stringify(result), { status: 201 });
```